docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -69,36 +98,75 @@ NOTE: The `keycloakConfig` field is partially implemented in link:{WFM-RC-CoreUR
                     "use-resource-role-mappings": true
                   }
 ----
+NOTE: Ensure that these values corresponds to your client configuration within your realm. For more information,
+see the {Keycloak} link:{WFM-RC-KeycloakURL}securing_apps/topics/oidc/nodejs-adapter.html[Node.js Adapter documentation].
+
+IMPORTANT: Filling in the values in the keycloakConfig also enables {Keycloak} in the demo server application
+
+[discrete]
+=== Keycloak Configuration in the Demo Mobile
+The _Keycloak Configuration_ for the demo mobile is already filled in. This may be customized by changing the contents of the
+application's link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/mobile/src/config.json[config.json]
```

Comment:
What contents. Mention that URL needs to be changed. IF they start to change stuff you will get support tickets! 
We do not support other changes until someone is aware what they are doing!
Documentation for this is essential to reference in case of some problems on the developers side.